utils/perf.html

```diff
@@ -35,14 +22,28 @@
   </thead>
   <tbody class="results"></tbody>
 </table>
+<!-- video.js -->
+<script src="../node_modules/video.js/video.dev.js"></script>
+
+<!-- HLS plugin -->
+<script src="../src/video-js-hls.js"></script>
+<script src="../src/flv-tag.js"></script>
+<script src="../src/exp-golomb.js"></script>
+<script src="../src/h264-stream.js"></script>
+<script src="../src/aac-stream.js"></script>
+<script src="../src/segment-parser.js"></script>
+
+<!-- MPEG-TS segment -->
+<script src="../test/data/ts-segment-bc.js"></script>
```

Comment:
I didn't even notice that! :) I was actually asking why the `<script>` elements moved from the top of the file to the bottom.